src/reason-parser/reason_lexer.mll

```diff
@@ -348,10 +348,13 @@ let preprocessor = ref None
 *)
 
 
+let latin1IdentifierMessage =
+  "ISO-Latin1 characters in identifiers - This error is being incorrectly reported as " ^
+  "'unused variable warning' but it's really an issue with the characters in your identifier. "
+
 let warn_latin1 lexbuf =
   Location.prerr_warning (Location.curr lexbuf)
-    (Warnings.Deprecated "ISO-Latin1 characters in identifiers")
-;;
+    (Warnings.Unused_var latin1IdentifierMessage)
```

Comment:
Why not just use `Location.deprecated` and some conditional compilation.
This is how the issue is handled in ppx_ast https://github.com/janestreet/ppx_ast/blob/master/src/jbuild#L27

Reply:
We'll probably do something like that eventually - I just wanted to quickly unblock 4.06. Pull requests welcome (but we're trying to avoid bash in the build (and remove any remaining bash-isms) so we'd need to tweak that example a bit).

Reply:
You don't need bash to replicate what's done in `ppx_ast`. It's just an ocaml program printing ocaml source code on stdout. 
I'll let you deal with people complaining about weird error messages ;)

Reply:
Yeah we will refine this. Thanks for the suggestion.